CI/travis/make_linux

```diff
@@ -21,15 +22,32 @@ handle_default() {
 	# set it back, to error immediately.
 	set -e
 
+	echo "### cmake -DENABLE_PACKAGING=ON -DDEB_DETECT_DEPENDENCIES=ON"
+	echo "          -DPYTHON_BINDINGS=ON ${DOC_FOUND}"
 	cmake -DENABLE_PACKAGING=ON -DDEB_DETECT_DEPENDENCIES=ON \
 		-DPYTHON_BINDINGS=ON ${DOC_FOUND} ..
 
+	echo "### make"
 	make
+	if [ "$TRAVIS" = "true" ] ; then
+		# make sure the *.so can be found during make package
+		echo "### Installing package as root"
+		sudo make install
+		# get rid of anything that will cause a problem
+		echo "### Deleting install_manifest.txt (created as root)"
+		sudo rm ./install_manifest.txt
+	else
+		echo "### fail"
+		printenv
```

Comment:
i would be careful with `printenv`
hopefully no secrets get exposed with this;

Reply:
yeah, I really only put that there for debugging - you are right - it should be removed.